src/v0/destinations/klaviyo/util.js

```diff
@@ -69,11 +68,6 @@ const getIdFromNewOrExistingProfile = async (endpoint, payload, requestOptions)
   let statusCode = resp.status;
   if (resp.status === 201 || resp.status === 409) {
     // retryable error if the profile id is not found in the response
-    errNotificationClient.notify(
-      new Error('Klaviyo: ProfileId not found'),
-      'Profile Id not Found in the response',
-      JSON.stringify(resp.response),
-    );
```

Comment:
Why we have removed this? Is it because the debugging we were doing for klaviyo and now it's not needed ?

Reply:
Yes